### ISSUE TITLE
A Resource to put logging configuration for a wafv2 webacl

### ISSUE THREAD

**Issue by itsSaad** [terraform-providers/terraform-provider-aws#13855]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
AWS supports updating of the logging for a WAFv2 WebACL with a `PutLoggingConfiguration` api call. More details here: https://docs.aws.amazon.com/waf/latest/APIReference/API_PutLoggingConfiguration.html

### New or Affected Resource(s)

* `aws_wafv2_webacl_logging_configuration`
or
* `logging_configuration` block inside `aws_wafv2_webacl` resource

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References
https://docs.aws.amazon.com/waf/latest/APIReference/API_PutLoggingConfiguration.html

* #0000

**Comment by adamhathcock** [comment]
I also need this before WAFv2 is usable for me.  Feels like an oversight.  Thanks!

**Comment by anGie44** [comment]
This feature has been merged and will release with `v2.68.0` of the Provider, expected to be out later today.
